src/helpers/themes/theming.css

```diff
@@ -36,17 +48,23 @@
 
   /* Ecosounds Colours */
   /* --oe-theme-hue: 207;
-    --oe-theme-saturation: 100%;
-    --oe-theme-lightness: 25%; */
+  --oe-theme-saturation: 100%;
+  --oe-theme-lightness: 25%; */
+
+  /* Microsite Colours */
+  /* --oe-theme-hue: 30deg;
+  --oe-theme-saturation: 62%;
+  --oe-theme-lightness: 49%; */
 
   --oe-background-color: hsl(var(--oe-theme-hue), var(--oe-theme-saturation), 100%);
   --oe-font-color: hsl(
     var(--oe-theme-hue),
     calc(var(--oe-theme-saturation) - 50%),
     calc(var(--oe-theme-lightness) - 90%)
   );
+  --oe-font-color-strong: color-mix(in oklch, var(--oe-primary-color), 80% black);
 
-  --oe-border-rounding: 0.5em;
+  --oe-border-rounding: 6px;
```

Comment:
i don't actually know what's best, but do we want border radius to not scale with font size?

i think we actually don't , like you have, but maybe we could look up some best practice

Reply:
This has been something I've been curious about for a while.

My initial thoughts are:

I suspect we want either `px` or `rem` units.
`em` units are definitely not what we want because `em` units are relative to the current font size, meaning that if we had larger font, the border rounding would increase!

I'll do some research into best practices and let you know what we want